modules/openapi-generator/src/main/resources/python/model_templates/method_init_composed.mustache

```diff
@@ -10,7 +10,52 @@
         '_additional_properties_model_instances',
     ])
 
-{{> model_templates/method_init_shared }}
+    @convert_js_args_to_python_args
+    def __init__(self, *args, **kwargs):  # noqa: E501
```

Comment:
requiredVars have been moved out of the init signature because the required vars were defined in composed oneOf/anyOf/allOf schemas and this schema does not know about them. Every schema is validated separately. So that means that for a composed schema with no `type` `required` or `properties` or `additionalProperties` definition inside it.
- all types are valid
- there are no required properties
- for object types, all properties are allowed because additionalProperties omitted defaults to additionalProperties: True which allows string property names and any type of value
- THEN the payload is checked against any oneOf/anyOf/allOf schemas

Reply:
This change will need to be made but it should be made in the future when I fix the value of vars and requiredVars to only include the info in each schema. Reverting this back to what it was before. That w ay these updates are incremental.